custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -248,19 +253,22 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
 
     function renderPercentageAndPartials(percentage, nominator, denominator, indicator) {
         if (isZeroNullUnassignedOrDataNotEntered(denominator)) {
-            return '<div> No expected ' + indicator + ' </div>';
+            return isMobile ? ('No expected ' + indicator) : '<div> No expected ' + indicator + ' </div>';
         }
         else {
             if (denominator === vm.dataNotEntered) { return vm.dataNotEntered; }
             if (percentage === vm.dataNotEntered) {
                 if (nominator === 0 && denominator === 0) {
-                    return '<div><span>100 %<br>(' + nominator + ' / ' + denominator + ')</span></div>';
+                    return isMobile ? ('100% (' + nominator + ' / ' + denominator + ')') :
```

Comment:
nit: not your fault but nominator --> numerator.

Reply:
That totally went over my head